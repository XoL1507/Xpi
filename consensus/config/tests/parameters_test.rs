// Copyright (c) Mysten Labs, Inc.
// SPDX-License-Identifier: Apache-2.0

use consensus_config::Parameters;
use insta::assert_yaml_snapshot;

#[test]
fn parameters_snapshot_matches() {
    let parameters = Parameters::default();
    assert_yaml_snapshot!("parameters", parameters)
}
