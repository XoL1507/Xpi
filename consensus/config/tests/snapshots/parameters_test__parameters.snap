---
source: consensus/config/tests/parameters_test.rs
expression: parameters
---
leader_timeout:
  secs: 0
  nanos: 250000000

